releases/release-1.31/release-notes/maps/pr-123669-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 123668
+releasenote:
+  text: 'cel: converting a quantity value into a quantity value failed.'
```

Comment:
I think this needs clarification. Is this a feature and is it supposed to fail?

Reply:
It looks like it's a one line bug fix: https://github.com/kubernetes/kubernetes/pull/123669/files (The type was just wrong)